opal/runtime/opal_params.c

```diff
@@ -61,6 +61,10 @@ char *opal_timing_output = NULL;
 bool opal_timing_overhead = true;
 #endif
 
+#if OPAL_ENABLE_DEBUG
```

Comment:
this variable wasn't conditionally included in master - may get some pushback here.   leave it for now.